x-pack/plugins/apm/public/components/app/ErrorGroupOverview/Watcher/WatcherFlyOut.js

```diff
@@ -228,7 +228,7 @@ export default class WatcherFlyout extends Component {
           read our{' '}
           <EuiLink
             target="_blank"
-            href={_.get(ELASTIC_DOCS, 'watcher-get-started.url')}
+            href={ELASTIC_DOCS['watcher-get-started'].url}
```

Comment:
No need to use `_.get` when it's a static object (when this file is typescript we'll get more safety)